composer.json

```diff
@@ -11,9 +11,9 @@
         }
     ],
     "require": {
-        "php": ">=5.3",
-        "symfony/framework-bundle": "2.*",
-        "symfony/symfony": "2.*"
+        "php": ">=5.5.9",
```

Comment:
This is not required for this bundle (for example, you can still use it with Symfony 2.8 that does not require PHP 5.5).